res/css/views/voip/_VideoLobby.scss

```diff
@@ -0,0 +1,173 @@
+            width: unset !important;
+            height: unset !important;
+            min-width: 0;
+            min-height: 0;
+            flex: 0 1 200px;
+        }
+
+        video {
+            position: absolute;
+            top: 0;
+            width: 100%;
+            height: 100%;
+            object-fit: cover;
+            display: block;
+            transform: scaleX(-1); // flip the image
+            background-color: black;
+        }
+
+        .mx_VideoLobby_controls {
+            position: absolute;
+            bottom: 0;
+            left: 0;
+            right: 0;
+
+            background-color: rgba($video-lobby-background, 0.9);
+
+            display: flex;
+            justify-content: center;
+            gap: $spacing-24;
+
+            .mx_VideoLobby_deviceButtonWrapper {
+                position: relative;
+                margin: 6px 0 10px;
+
+                .mx_VideoLobby_deviceButton {
+                    $size: 50px;
+
+                    width: $size;
+                    height: $size;
+
+                    background-color: $video-lobby-primary-content;
+                    border-radius: calc($size / 2);
+
+                    &::before {
+                        content: '';
+                        display: inline-block;
+                        mask-repeat: no-repeat;
+                        mask-size: 20px;
+                        mask-position: center;
+                        background-color: $video-lobby-system;
+                        height: 100%;
+                        width: 100%;
+                    }
+
+                    &.mx_VideoLobby_deviceButton_audio::before {
+                        mask-image: url('$(res)/img/voip/call-view/mic-off.svg');
+                    }
+
+                    &.mx_VideoLobby_deviceButton_video::before {
+                        mask-image: url('$(res)/img/voip/call-view/cam-off.svg');
+                    }
+                }
+
+                .mx_VideoLobby_deviceListButton {
+                    $size: 15px;
+
+                    position: absolute;
+                    bottom: 0;
+                    right: -2.5px;
+                    width: $size;
+                    height: $size;
+
+                    background-color: $video-lobby-primary-content;
+                    border-radius: calc($size / 2);
+
+                    &::before {
+                        content: '';
+                        display: inline-block;
+                        mask-image: url('$(res)/img/feather-customised/chevron-down.svg');
+                        mask-size: $size;
+                        mask-position: center;
+                        background-color: $video-lobby-system;
+                        height: 100%;
+                        width: 100%;
+                    }
+                }
+
+                &.mx_VideoLobby_deviceButtonWrapper_active {
+                    .mx_VideoLobby_deviceButton, .mx_VideoLobby_deviceListButton {
```

Comment:
Should this not be broken into two lines according to https://github.com/vector-im/element-meta/blob/travis/temp-code-style-web/web-code-style.md#scss (no.6) ?